game/scripts/npc/items/item_bloodstone.txt

```diff
@@ -85,7 +85,7 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "mana_regen_multiplier"                           "150 200 350 500 750"
+        "mana_regen_multiplier"                           "200 250 300 350 400"
```

Comment:
I think while storm nerf's are justified nerfing bloodstone could hurt some other heroes. If we're going to be conservative before tournament I probably wouldn't nerf late game bloodstone by over 300%. I think 200 275 350 425 500 would be a fairer nerf. I also don't think storm's mana is entirely the issue and would probably increase the unit interval for his talent as well from like 300 to 500 or 400.

Reply:
or 150 250 350 450 550

Reply:
I dont think any other heroes actually have mana issues after a bloodstone apart from storm

Reply:
does chatterjee still have mana issues or did they fix her up a bit?

Reply:
The nerf on bloodstone is harsh because mana regen amp stacks. And I don't want BTS to see Storm Spirit with multiple bloodstones on their stream. Other heroes don't suffer from the nerf (not even Chatterjee).

Reply:
I honestly think Storm Spirit should just be removed from the game, along with Tinker. These heroes are pretty much 100% balanced around mana pool and I personally don't think there is much way for these heroes to be balanced without having unique limiters on their heroes.

Reply:
One hero shouldn't have to be the reason why bloodstone has to be nerfed. But Because this probably won't happen I have some other suggestions:

- Change Storm spirits level 25 talent 550 auto remnant to another talent (maybe +40 all stats or something.) 

- Make Bloodstones Not stack with eachother (Valve may do this but stacking bloodstones is impractical in normal dota). Storm spirit can get +300 Mana regen in normal dota just by stacking bloodstones, and that is enough for it to be stupid. And In OAA farming up 4/5 bloodstones is easy.

Reply:
Ok, I forgot to say this: This PR is just a temporary solution.

Reply:
ok